src/main/java/parser/Parser.java

```diff
@@ -1,17 +1,6 @@
 package parser;
 
-import command.Command;
-import command.AddCommand;
-import command.ClearCommand;
-import command.DeleteCommand;
-import command.ExitCommand;
-import command.ListCommand;
-import command.FindCommand;
-import command.HelpCommand;
-import command.ViewCommand;
-import command.FavouriteCommand;
-import command.SettingsCommand;
-import command.CommandEnum;
+import command.*;
```

Comment:
I'm not really sure if they would be nitpicky on the use of '*', I've been trying to avoid it because of that.

Reply:
Did you just list it one by one?

Reply:
i think so but it's really ugly and I hate it